index.html

```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
+                <li><a href="#">Ruby</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">CSS</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec2.svg" alt="rec" />
+              <h4>Frameworks</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">React.js</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSpec</a></li>
+                <li><a href="#">Capybara</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec3.svg" alt="rec" />
+              <h4>skills</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">Database Management</a></li>
+                <li><a href="#">Version Control</a></li>
+                <li><a href="#">CLI</a></li>
+                <li><a href="#">API Design</a></li>
+                <li><a href="#">Web Design</a></li>
+              </ul>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="interest">
+        <div class="intrest-description">
+          <h5>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <div class="contact">
+          <div class="contact-input">
+            <input type="text" placeholder="Full Name" name="name" required />
+          </div>
+          <div class="contact-input">
+            <input
+              type="email"
+              placeholder="Email Address"
+              name="email"
+              required
+            />
+          </div>
+          <div class="info">
+            <textarea placeholder="Write something ..." rows="6"></textarea>
+          </div>
+        </div>
+        <div class="contact-btn">
+          <div class="story-btn">
+            <span><a href="#">Get in touch</a></span>
+          </div>
+        </div>
+      </div>
     </section>
   </body>
+  <footer>
+    <div class="footer-content">
+      <ul class="social-media footer-social">
+        <li>
+          <a href="#"><img src="./pics/github.svg" alt="github" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/link.svg" alt="linkedin" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/yah.svg" alt="yep" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/twitter.svg" alt="twitter" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/medium.svg" alt="medium" /></a>
+        </li>
+      </ul>
+      <hr />
+    </div>
+  </footer>
```

Comment:
Please make sure to use the same colors in the template the background of this footer is different 
![temp2 figma footer](https://user-images.githubusercontent.com/59707859/228217295-bb144526-9582-4ae4-8231-fe2935d8d3df.PNG)
![temp2 student footer](https://user-images.githubusercontent.com/59707859/228217300-80a939b0-ac50-4f5b-9bca-d0146f9d310d.PNG)

Reply:
done :heavy_check_mark: